resources/public/api/conf/1.0/codingOutUnderpayments.yaml

```diff
@@ -0,0 +1,27 @@
+get:
+  summary: Retrieve Coding Out Underpayments and Debt Amounts [test only]
+
+  security:
+    - User-Restricted:
+        - read:self-assessment
+  $ref: './retrieveCodingOutUnderpayments.yaml'
+
+  parameters:
+    - $ref: './common/queryParameters.yaml#/components/parameters/source'
+
+put:
+  summary: Create or Amend Coding Out Underpayments and Debt Amounts [test only]
+
+  security:
+    - User-Restricted:
+        - write:self-assessment
+
+  $ref: './createOrAmendCodingOutUnderpayments.yaml'
+
+delete:
+  summary: Delete Coding Out Underpayments and Debt Amounts [test only]
+
+  security:
+    - User-Restricted:
+        - write:self-assessment
+  $ref: './deleteCodingOutUnderpayments.yaml'
```

Comment:
None of these render on the page atm (see pic).  Not sure why as the yaml appears to be valid...
<img width="1436" alt="image" src="https://user-images.githubusercontent.com/106095575/215817444-ef042617-e1da-49c4-ae44-cd1138550a3e.png">

Reply:
These endpoints still aren't appearing for me